SCClassLibrary/Platform/Platform.sc

```diff
@@ -91,9 +91,12 @@ Platform {
 		}
 	}
 
-	loadStartupFiles { this.startupFiles.do{|afile|
-		afile = afile.standardizePath;
-		if(File.exists(afile), {afile.load})
+	loadStartupFiles {
```

Comment:
ok, that is a way to do it. 

Maybe it would be easier to understand and remember if the local case was completely analogous to the global case, i.e. you could have a startup file local to the config file?